### PR TITLE
Adds AI Dashboard Generator Readme

### PR DESCRIPTION
**Pull request by EstaniPP**
PR to add a new readme.md file and a MIT open source license. Both of the files were created by ChatGPT

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,47 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# AI Dashboard Generator
 
-## Getting Started
+AI Dashboard Generator is an open-source dashboard generator based on Next.js and ChatGPT. The tool allows users to feed existing tables or datasets, and from there, it understands the intent of the table and generates a visual representation of the data.
```
Comment (apresmoi) [inline README.md]: Instead of "ChatGPT" it should be "OpenAI's GPT3 technology". Since we are using the GPT3 API instead of the Chat.

File: README.md [hunk 2]
```diff
@@ -1,36 +1,47 @@
...
 
-First, run the development server:
+Internally, AI Dashboard Generator uses a custom CSV dataset and ChatGPT with a prompt model contextualized around JSON responses to give simple and useful visualizations. You can filter the parameters and export the dataviz directly into javascript code.
```
Comment (apresmoi) [inline README.md]: I would rephrase this. Since the dashboard has those CSV datasets as examples so the user is not welcomed by a empty screen.
Also, same as before, we are not using the Chat.

Maybe something like "The Dashboard Generator uses a prompt engineered to make the AI reply using JSON to create simple and useful visualizaitons. We do not add any requirements for the titles or content, the AI does it all by using the data...."

Explaining a little bit that we are not actually hardcoding anything about the data,  just the sort of responses that we want.

File: README.md [hunk 3]
```diff
@@ -1,36 +1,47 @@
...
-```bash
-npm run dev
-# or
-yarn dev
+## Installation
+
+To get started with the AI Dashboard Generator, you'll need to have Node.js and npm installed on your machine. Once you've done that, you can install the dependencies by running the following command in your terminal:
+
+```
+> npm install
```
Comment (apresmoi) [inline README.md:14]: Maybe we should add here or in another sections the instructions on how to run the code after it's installed?

**Review by apresmoi — CHANGES_REQUESTED** [review]
Hey @EstaniPP ! Thank you for the PR.

It looks ok, I added some comments to improve the text in some of the sections.

**Comment by EstaniPP** [comment]
@apresmoi thanks for the review! I made requested changes, please tell me how you feel with them.

**Comment by apresmoi** [comment]
The changes look fine @EstaniPP, thank you!
